data/mods/Magiclysm/snippets/survivor_notes.json

```diff
@@ -8,5 +8,15 @@
         "text": "\"(Spell Name) RECIPE V3\n---\nRare Ingredients:\n- 17 drams owl's blood - less no effect, more turns bear into large owl\n- 1 gastrolith, large american alligator\n- 3 teeth or claws from a honey badger\n- 8 pounds bear meat, preferably polar bear, grizzly will suffice\nBring a large pot (meat must fully submerge) filled about half way with potion starter fluid (high grade) to a boil.  Add gastrolith and teeth/claws after boiling, add the blood, stir counterclockwise for a minute while chanting standard polymorph spell.  Continue until concoction has a slight glow.  Add the bear meat, remove from heat, cover for 24 hours.\nBear MUST eat all 8 pounds of the meat (claws and gastrolith optional).\n---\n*I've finally gotten my hands on enough polar bear meat.  The grizzly variations are a bit lackluster.*\nThere are flecks of blood on the page.\""
       }
     ]
+  },
+  {
+    "type": "snippet",
+    "category": "note",
+    "text": [
+      {
+        "id": "magiclysm_note_2",
+        "text": "\"Musings on Orichalcum\n\nOrichalcum is a primarily copper based alloy, but the creation varies from person to person in terms of minor metals and exact procedure. While not substantially stronger than steel, it's far more resistant to corrosion and general wear (some say it has a will to remain in it's shape). While conventional ancient magical swords of power exist, a lot of the common understanding is based on pre-industrial people finding orichalcum weaponry in tombs and such in working condition. So why isn't orichalcum used in industrial contexts? The issue is that the magic can start to degrade if not forged with proper skill. For most failures it will revert in terms of minutes (never being orichalcum in practice to begin with), but there is a notable risk of failure up to 30 years, meaning that objects has to be \"aged\" like wine to be considered true orichalcum. Although slightly easier than forging it from scratch, reforging existing orichalcum poses the same problems, thus blocking industrial manufacturing and limiting it to craft use. You can find it in old armor and weaponry, but also in expensive heirloom cutlery and such. There is still an active community of hobbyists and artisan experts, but orichalcum is like many other magical phenomena not suited for our modern society at large.\""
```

Comment:
```suggestion
        "text": "\"Musings on Orichalcum\n\nOrichalcum is a primarily copper based alloy, but the creation varies from person to person in terms of minor metals and exact procedure.  While not substantially stronger than steel, it's far more resistant to corrosion and general wear (some say it has a will to remain in its shape).  While conventional ancient magical swords of power exist, a lot of the common understanding is based on pre-industrial people finding orichalcum weaponry in tombs and such in working condition.  So why isn't orichalcum used in industrial contexts?  The issue is that the magic can start to degrade if not forged with proper skill.  For most failures it will revert in terms of minutes (never being orichalcum in practice to begin with), but there is a notable risk of failure up to 30 years, meaning that objects have to be \"aged\" like wine to be considered true orichalcum.  Although slightly easier than forging it from scratch, reforging existing orichalcum poses the same problems, thus blocking industrial manufacturing and limiting it to craft use.  You can find it in old armor and weaponry, but also in expensive heirloom cutlery and such.  There is still an active community of hobbyists and artisan experts, but orichalcum is like many other magical phenomena not suited for our modern society at large.\""
```